### PR TITLE
Navigation titles

### PR DESCRIPTION
**Pull request by jamietsao**
https://github.com/jamietsao/codepath-mapstagram/issues/106

### CODE REVIEW

File: Mapstagram/ViewControllers/MapViewController.swift [hunk 1]
```diff
@@ -128,11 +127,15 @@ class MapViewController: MediaViewController {
                 SessionService.getInstance().setAllViewsToNeedReloading()
                 media = Array(media[1..<media.count])
                 currMedia = currMedia + media
+                SessionService.getInstance().setMediaArray(currMedia)
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/MapViewController.swift]: I reordered lines of code here to match the order in `reloadMediaData()`, and as mentioned in the TODO below, we should refactor this as it duplicates most of what's in there.

**Comment by jamietsao** [comment]
Pretty cool to see location names in the nav bar.  Definitely helps with usability.

File: Mapstagram/ViewControllers/MapViewController.swift (hunk 1 above)
Comment (kristen) [inline Mapstagram/ViewControllers/MapViewController.swift]: Thank you for doing this.

**Comment by kristen** [comment]
I think it's good. I can't wait to see it. Merge it!

**Comment by jamietsao** [comment]
Merged!  It's pretty cool when you search for a specific place, like "Disneyland".  Man, I can't wait to come up with a cool demo.  =)

**Pull request merged**